test/fuzzTesting/createRandomStateTest.cpp

```diff
@@ -147,16 +100,16 @@ int main(int argc, char *argv[])
 	read_string(s, v);
 
 	// insert new random code
-	v.get_obj().find("randomStatetest")->second.get_obj().find("pre")->second.get_obj().begin()->second.get_obj()["code"] = "0x" + randomCode + (randGen() > 128 ? "55" : "") + (randGen() > 128 ? "60005155" : "");
+	v.get_obj().find("randomStatetest")->second.get_obj().find("pre")->second.get_obj().begin()->second.get_obj()["code"] = "0x" + randomCode;
```

Comment:
the reason I added the ...55 opcode at the end, was to write values from the stack or from memory to the storage so this can be tested. Very useful.